packages/plugin-rsc/src/css-query.test.ts

```diff
@@ -0,0 +1,76 @@
+import { describe, expect, test } from 'vitest'
+
+/**
+ * Check if a CSS import ID has special queries that should be excluded from CSS collection.
+ * These queries transform CSS imports to return different data types rather than actual CSS to be linked.
+ */
+function hasSpecialCssQuery(id: string): boolean {
```

Comment:
remove trivial unit tests. add e2e instead by updating `examples/basic` and `e2e/basic.test.ts`.